### PR TITLE
Added problem statement, project goal, 10x overview, and user groups

### PR DESCRIPTION
**Pull request by willcahoe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,31 @@
-# 10x
+# 10x website Problem Statement and Goals
+
+### [Macro]
+
+**Opportunity:** The US Government employs many capable public servants who observe problems with government technology and are ideally positioned to pursue solutions to these problems.
```
Comment (carodew) [inline README.md]: replace`US Government` with `U.S. government` per https://content-guide.18f.gov/specific-words-and-phrases/

File: README.md [hunk 2]
```diff
@@ -1,6 +1,31 @@
-# 10x
+# 10x website Problem Statement and Goals
+
+### [Macro]
```
Comment (carodew) [inline README.md]: Heading levels should kind of act like an outline indent and shouldn't skip levels. That is, each page should have one h1, and the next level down should be an h2 (or two ## in markdown). 

So either change the Macro header to be H2, or make it a subheader in a section that starts with an h2. 

Also stylistically I don't think we need the square brackets, it looks a little strange.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,31 @@
...
+
+**Opportunity:** The US Government employs many capable public servants who observe problems with government technology and are ideally positioned to pursue solutions to these problems. 
+
+**Problem:** Unfortunately, many of these talented folks are unable to pursue solutions to these problems due to a lack of funding, support, or necessary skills. 
+
+### [Micro]
```
Comment (carodew) [inline README.md]: Per above, make h2 instead of h3.

File: README.md [hunk 4]
```diff
@@ -1,6 +1,31 @@
...
+### [Micro] 
+
+**Opportunity:** The 10x team can offer funding and guidance to creative civil servants who want to solve problems facing the government in technology  spaces.
+
+**Problem:** Most civil servants who might benefit from 10x funding, guidance, or investment model do not know we exist.
+
+## About 10x 
+
+10x is an incremental investment pipeline that lives in GSA’s Technology Transformation Services’ (TTS) Office of Products and Programs (OPP). We fund products and services that possess groundbreaking ideas or technology with the potential for significant
+return on investment. Our approach is based on modern venture capital practices and  is  designed to spend fewer taxpayer dollars while also ensuring that the best products and services get funded.
```
Comment (carodew) [inline README.md]: remove line break

Comment (carodew) [reply README.md]: The last sentence feels like a run on sentence, I'm wondering if we can put a comma after `taxpayer dollars` to help with that? 

Also it appears that there are two spaces around `is` in `capital practices and is designed to spend`

File: README.md [hunk 5]
```diff
@@ -1,6 +1,31 @@
...
+
+## Website goal
+
+> The 10x website will provide civil servants across government the information they need to apply for funding and guidance to explore their ideas to solve government problems. It will also provide information and resources to civil servants who want to model something similar within their own agencies, as well as up-to-date financial information for GSA leadership and oversight bodies. 
+
+## Expected user groups
```
Comment (carodew) [inline README.md]: I'm not sure we need the word `expected` -- it sounds timid.

Comment (carodew) [reply README.md]: It appears there's an indent on line 22, which I believe marks it up as a pull quote. Semantically, that doesn't really make sense here. I'd remove the indent and leave it as a regular paragraph.

File: README.md (hunk 4 above)
Comment (willcahoe) [reply README.md]: Reworded a bit

**Pull request closed without merging**